terraform/implementation/setup/terraform.tfvars.example

```diff
@@ -0,0 +1,12 @@
+/*
+* Terraform variables file, copy this file to <ENVIRONMENT>.tfvars ( development.tfvars, production.tfvars ) and fill in the values you require.
+* Remove the comments and replace the values with the desired values for your environment.
+* This tfvars file is used to configure the environment for the Terraform implementation.
+* The 'owner' variable specifies the owner of the resources that will be provisioned and is included in resource naming and tagging.
+* The 'region' variable specifies the AWS region where the resources will be provisioned.
+* The new file you create will not be tracked in git by default, so make sure you store it in a secure location for later use.
+* required:
+* optional:
+*/
+owner = "JurrasicPark"
```

Comment:
`JurassicPark` 😄

Reply:
I fixed the spelling, thanks :sauropod: :t-rex: